src/resources/UsageAnalytics/Read/Dimensions/DimensionsInterfaces.ts

```diff
@@ -33,6 +33,10 @@ export interface DimensionModel {
      * The type of the dimension
      */
     type: DimensionType;
+    /**
+     * Strings that correspond to the path that must be followed to fetch the desired value of the dimension in the custom_datas map.
```

Comment:
Explanation from the [documentation](https://docs.coveo.com/en/17/cloud-v2-api-reference/usage-analytics-read-api#operation/get__v15_dimensions_custom):
```suggestion
     * The path in the custom_datas map corresponding to this dimension. For custom dimensions only.
```